frontend/src/GlobalStyle.js

```diff
@@ -20,7 +20,7 @@ const GlobalStyle = createGlobalStyle`
     text-align: inherit;
     background: transparent;
     color: inherit;
-    font-family: helvetica;
+    font-family: 'helvetica';
```

Comment:
First time seeing this. You normally see a range of fonts in a definition, not just one. Probably should fill this out more.